### ISSUE TITLE
How can I use as Second Screen?

### ISSUE THREAD

**Issue by vfransant** [kbumsik/VirtScreen#72]
The virtscreen is just mirrorring my laptop screen, and I'd like to use it as a second screen in my iPad. What do I have to do?

**Comment by xentipod** [comment]
Open arandr through the terminal, and you should be able to move the windows around to your liking.

**Comment by vfransant** [comment]
Thanks for your help!

**Comment by xentipod** [comment]
> Thanks for your help!

Did it work? If so, no problem.

**Comment by x-dongzi** [comment]
How to open with paramter arandr?
Below is the output in my terminal.

virtscreen --right --log=INFO --arandr
usage: virtscreen [-h] [--auto] [--left] [--right] [--above] [--below]
                  [--portrait] [--hidpi] [--log LOG]
virtscreen: error: unrecognized arguments: --arandr

**Comment by xentipod** [comment]
@x-dongzi 
I open/started Virtscreen through the GUI (so by launching virtscreen and clicking "enable virtual screen"). I then opened arandr in the terminal by running `arandr`. This should open arandr, and allow you to move your virtual screens.

Note: You need to have arandr installed

**Comment by x-dongzi** [comment]
> @x-dongzi
> I open/started Virtscreen through the GUI (so by launching virtscreen and clicking "enable virtual screen"). I then opened arandr in the terminal by running `arandr`. This should open arandr, and allow you to move your virtual screens.
> 
> Note: You need to have arandr installed

Thanks. I could run arandr now. But it's very strange on my laptop. My OS is deepin which is based in debian.
I tried to start it in CLI mode:
virtscreen --right --log=INFO
The second screen in vncviewer is black screen. The mouse can be moved to right second screen. But If I drag any windows to the right screen. The second screen is always black and doesn't show anything.
Do you know how I could debug or resolve this problem?
![2020-12-05_12-35](https://user-images.githubusercontent.com/1109099/101233945-63412300-36f6-11eb-97fe-e45dd25726f7.png)
I also take a photo to show the problem.
![微信图片_20201205124123](https://user-images.githubusercontent.com/1109099/101234039-36d9d680-36f7-11eb-80c6-020dc401ba0b.jpg)


The output of xrandr -q is below:
dong@dong-PC:~/Desktop$ xrandr -q
Screen 0: minimum 320 x 200, current 2390 x 768, maximum 8192 x 8192
LVDS-1 connected primary 1366x768+0+0 (normal left inverted right x axis y axis) 0mm x 0mm
   1366x768      60.17*+
   1360x768      59.80    59.96  
   1280x720      60.00    59.99    59.86    59.74  
   1024x768      60.04    60.00  
   960x720       60.00  
   928x696       60.05  
   896x672       60.01  
   1024x576      59.95    59.96    59.90    59.82  
   960x600       59.93    60.00  
   960x540       59.96    59.99    59.63    59.82  
   800x600       60.00    60.32    56.25  
   840x525       60.01    59.88  
   864x486       59.92    59.57  
   800x512       60.17  
   700x525       59.98  
   800x450       59.95    59.82  
   640x512       60.02  
   720x450       59.89  
   700x450       59.96    59.88  
   640x480       60.00    59.94  
   720x405       59.51    58.99  
   684x384       59.88    59.85  
   680x384       59.80    59.96  
   640x400       59.88    59.98  
   576x432       60.06  
   640x360       59.86    59.83    59.84    59.32  
   512x384       60.00  
   512x288       60.00    59.92  
   480x270       59.63    59.82  
   400x300       60.32    56.34  
   432x243       59.92    59.57  
   320x240       60.05  
   360x202       59.51    59.13  
   320x180       59.84    59.32  
VGA-1 disconnected 1024x768+1366+0 (normal left inverted right x axis y axis) 0mm x 0mm
   1510x2732_virt  59.98  
   1024x768_virt  59.92* 
HDMI-1 disconnected (normal left inverted right x axis y axis)
DP-1 disconnected (normal left inverted right x axis y axis)
HDMI-2 disconnected (normal left inverted right x axis y axis)
HDMI-3 disconnected (normal left inverted right x axis y axis)
DP-2 disconnected (normal left inverted right x axis y axis)
DP-3 disconnected (normal left inverted right x axis y axis)
  755x1354_virt (0x16d) 86.750MHz -HSync +VSync
        h: width   760 start  816 end  896 total 1032 skew    0 clock  84.06KHz
        v: height 1354 start 1357 end 1367 total 1404           clock  59.87Hz
  1354x755_virt (0x186) 83.500MHz -HSync +VSync
        h: width  1360 start 1432 end 1568 total 1776 skew    0 clock  47.02KHz
        v: height  755 start  758 end  768 total  784           clock  59.97Hz
  755x1366_virt (0x18d) 87.500MHz -HSync +VSync
        h: width   760 start  816 end  896 total 1032 skew    0 clock  84.79KHz
        v: height 1366 start 1369 end 1379 total 1416           clock  59.88Hz
  1366x755_virt (0x18e) 83.750MHz -HSync +VSync
        h: width  1368 start 1440 end 1576 total 1784 skew    0 clock  46.95KHz
        v: height  755 start  758 end  768 total  784           clock  59.88Hz

**Comment by xentipod** [comment]
@x-dongzi 
Yeah, I have the same issue. I think it might be a problem with the window manager, or something. I'm trying to find solutions to this problem, but haven't found any yet. I'll let you know if I find anything.
Do you happen to be using an amd gpu?

(Also, sorry about the late reply. Time zones are weird)
